change/@fluentui-react-tree-b6607987-a159-4abc-be0c-225294e9e8a7.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32442&commitId=5118d249bb4fd91057c6ab348ebe7bdb872363cc&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|5| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|5| Changed|

</details>


  <details><summary><b>Drawer</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay RTL.chromium.png|3282| Changed|
|Drawer.Full Overlay Dark Mode.chromium.png|983| Changed|
|Drawer.overlay drawer full.chromium.png|3285| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>